pkg/monitoring/domainstats/prometheus/prometheus_test.go

```diff
@@ -149,6 +149,31 @@ var _ = Describe("Prometheus", func() {
 			Expect(dto.Gauge.GetValue()).To(BeEquivalentTo(float64(1024)))
 		})
 
+		It("should send cached memory", func() {
+			ch := make(chan prometheus.Metric, 1)
+			defer close(ch)
+
+			ps := prometheusScraper{ch: ch}
+
+			domainStats := &stats.DomainStats{
+				Cpu: &stats.DomainStatsCPU{},
+				Memory: &stats.DomainStatsMemory{
+					CachedSet: true,
+					Cached:    1,
+				},
+			}
+			vmi := k6tv1.VirtualMachineInstance{}
+			ps.Report("test", &vmi, newVmStats(domainStats, nil))
+
+			result := <-ch
+			dto := &io_prometheus_client.Metric{}
+			result.Write(dto)
+
+			Expect(result).ToNot(BeNil())
+			Expect(result.Desc().String()).To(ContainSubstring("kubevirt_vmi_memory_cached_bytes"))
+			Expect(dto.Gauge.GetValue()).To(BeEquivalentTo(float64(1024)))
```

Comment:
Is there any way to provide more information about why this should be 1024? is this OS-agnostic? what if we run this test on different OSses?

Reply:
I think it's OS-agnostic, as we do a kibibytes conversion (and not kilobytes) by: https://github.com/kubevirt/kubevirt/blob/49d2689c779afd6f40371b672d445aeb5f76a14c/pkg/monitoring/domainstats/prometheus/prometheus.go#L158

In this unit test we do `report()` -> `updateMetrics()` -> `updateMemory()` which leads to this `*1024` conversion, but please correct me if I'm missing something.